docs/_config.yml

```diff
@@ -8,7 +8,7 @@ kramdown:
 ALLUXIO_RELEASED_VERSION: 1.2.0
 # This would be "master" for the master branch, 1.1.1 for the 1.1.1 release, and
 # 1.1.1-RC2 for the second 1.1.1 release candidate.
```

Comment:
Sounds good. I'll test the org site docs build and if builds correctly, will submit a separate PR if that works for you.